js/waiting.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+  var s = (new Date()).getTime();
+  var tmp = new moment.duration(moment(new Date(launchDate).getTime())-s);
+  var countdownStr = function(tms) {
+    return tms.days() + 'd ' + tms.hours() + 'h ' + tms.minutes() + 'm ' + (tms.seconds() <= 9 ? '0' + tms.seconds() : tms.seconds()) + 's';
+  };
+  $('#countdown').html(countdownStr(tmp));
+  $.ajax({
+    url: 'time.php',
+    data: {
+      t: (new Date()).getTime()
+    },
+    success: function(resp) {
+      var e = (new Date()).getTime();
+      var h = JSON.parse(resp);
+      var u = (new Date(h.month + ' ' + h.mday + ', ' + h.year + ' ' + h.hours + ':' + h.minutes + ':' + h.seconds)).getTime() - (e - s);
+      var i = (new Date(launchDate).getTime());
+      if(u > i) {
```

Comment:
Same here. I'm all for short variable names when the context makes it clear what they are, but I find this a little hard to follow.

Reply:
it's a nod to hui's style of coding back in the day.  i'll change it

Reply:
addressed in commit: c1b63e5b775889446974e4509b87fa5a2dd2b7cb

sorry @hshaosf